sky/cli.py

```diff
@@ -2100,6 +2112,9 @@ def storage_delete(names: Tuple[str], all: bool):  # pylint: disable=redefined-b
         # Delete two storage objects.
         sky storage delete imagenet cifar10
         \b
+        # Delete all storage objects matching glob pattern 'imagenet*'.
+        sky storage delete "imagenet*"
```

Comment:
The quotation marks are needed around "*" are needed, otherwise this will fetch the files in whichever current directory you are running from and not the storage object names.